jsk_fetch_robot/README.md

```diff
@@ -114,12 +127,48 @@ fetch-interface function APIs
   ```
 (send *ri* :start-grasp)
   ```
+  
   to release object
-
   ```
 (send *ri* :stop-grasp)
   ```
+  
+- inverse kinematics
 
+  including torso movement
+  ```
+(send *fetch* :inverse-kinematics (make-coords :pos #f(800 0 1200)) :debug-view t)
+  ```
+  
+  using only arm movement
+  ```
+(send *fetch* :rarm :inverse-kinematics (make-coords :pos #f(800 0 1200)) :degug-view t)
+  ```
+  
+  coordinates can be made with 
+  ```
+  (make-coords :pos #f(0 0 0) :rpy (float-vector 0 0 pi))
+  ```
+  
+- moving the robot
+ 
+ move 1 [m] forward, based on relative position (CAUTION, this will move the real robot)
+ ```
+ send *ri* :go-pos 1 0 0
```

Comment:
@Affonso-Gui Hi, thank you for great contribution. It is better not to omit toplevel parentheses for documentation.